roles/varnish/templates/etc/default/varnish

```diff
@@ -0,0 +1,10 @@
+START=yes
+NFILES={{ varnish.nfiles|default(131072) }}
+MEMLOCK={{ varnish.memlock|default(82000) }}
+DAEMON_OPTS="-j unix,user=vcache -F \
+             -a :{{ varnish.port }} \
+             -T 127.0.0.1:6082 \
+             -f /etc/varnish/varnish.vcl \
+             -S /etc/varnish/secret \
+             -s malloc,{{ varnish.malloc|default('16G') }} \
```

Comment:
Perhaps we should make these default `256m`, which is the default on install. It seems the reversal of what we have been doing, which was leave it low and then set it high when needed.

Looks like you are applying this default twice too.